pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/HashUtils.java

```diff
@@ -36,6 +40,31 @@ public static byte[] hashMD5(byte[] bytes) {
     return Hashing.md5().hashBytes(bytes).asBytes();
   }
 
+  /**
+   * Returns a byte array that is a concatenation of the binary representation of each of the passed UUID values. This
+   * is done by getting a String from each value by calling {@link Object#toString()}, which is then used to create a
+   * {@link UUID} object. The 16 bytes of each UUID are appended to a buffer which is then returned in the result.
+   * If any of the values is not a valid UUID, then we return the result of {@link #concatenate}.
+   */
+  public static byte[] hashUUID(Object[] values) {
```

Comment:
For clarification, we are not really hashing the values, but compress it instead right?